packages/beacon-node/src/chain/archiveStore/archiveStore.ts

```diff
@@ -34,6 +34,7 @@ export class ArchiverStore {
 
     if (!opts.disableArchiveOnCheckpoint) return;
 
+    // Observers are run in order these are registered to emitter
```

Comment:
Lets get this to stand out a bit more with a multi-line comment style
```suggestion
    /**
      * Observers are run in order these are registered to emitter
      */
```